staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/types.go

```diff
@@ -67,5 +69,24 @@ type Key struct {
 	Secret string `json:"secret"`
 }
 
+// GKMSConfig contains API configuration for Google KMS transformer.
+type GKMSConfig struct {
+	// projectID is the GCP project which hosts the key to be used. It is automatically set
+	// to your project if you are running Kubernetes on GKE/GCE.
```

Comment:
It would not. This field overrides defaults if it is set.
GKE does not set it at all, since it can be retrieved via the GCECloud object. There is no need to set it on GCE as well, because the cloud provider would provide it.

But if you do set it, it would override the project name provided by GCE. Will reword.